[BUG]:

### 报告清单

- [X] 我已仔细阅读并了解上述注意事项。
- [X] 我已使用最新版本测试过，确认问题依旧存在。
- [X] 我确定在 [GitHub Issues](../issues) 中没有相同或相似的问题。
- [ ] 我有足够的时间和能力，愿意为此提交 PR 来修复问题。

### 影响版本

现版本

### 问题描述

使用功能出错

### 复现步骤

02-11 02:18:46 [ERROR] nonebot | Running Matcher(type='message', module=nonebot_plugin_moyu) failed.
Traceback (most recent call last):
  File "<string>", line 17, in <module>
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\__init__.py", line 273, in run
    get_driver().run(*args, **kwargs)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\drivers\fastapi.py", line 187, in run
    uvicorn.run(
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\uvicorn\main.py", line 569, in run
    server.run()
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\uvicorn\server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "C:\Users\hlb\AppData\Local\Programs\Python\Python311\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
  File "C:\Users\hlb\AppData\Local\Programs\Python\Python311\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
  File "C:\Users\hlb\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 640, in run_until_complete
    self.run_forever()
  File "C:\Users\hlb\AppData\Local\Programs\Python\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Users\hlb\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Users\hlb\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 1919, in _run_once
    handle._run()
  File "C:\Users\hlb\AppData\Local\Programs\Python\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\message.py", line 141, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
> File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\message.py", line 187, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\internal\matcher\matcher.py", line 732, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\internal\matcher\matcher.py", line 707, in simple_run
    await handler(
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\dependencies\__init__.py", line 108, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot_plugin_moyu\__init__.py", line 111, in moyu
    await matcher.finish(MessageSegment.image(moyu_img))
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\internal\matcher\matcher.py", line 481, in finish
    await cls.send(message, **kwargs)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\internal\matcher\matcher.py", line 466, in send
    return await bot.send(event=event, message=_message, **kwargs)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot_plugin_guild_patch\__init__.py", line 19, in patched_send
    return await original_send(self, event, message, **kwargs)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\adapters\onebot\v11\bot.py", line 220, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\adapters\onebot\v11\bot.py", line 174, in send
    return await bot.send_msg(**params)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\internal\adapter\bot.py", line 120, in call_api
    raise exception
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\internal\adapter\bot.py", line 98, in call_api
    result = await self.adapter._call_api(self, api, **data)
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\adapters\onebot\v11\adapter.py", line 132, in _call_api
    return handle_api_result(await self._result_store.fetch(seq, timeout))
  File "F:\源代码\PY\noneBot\BotOne\.venv\Lib\site-packages\nonebot\adapters\onebot\v11\utils.py", line 58, in handle_api_result
    raise ActionFailed(**result)
nonebot.adapters.onebot.v11.exception.ActionFailed: ActionFailed(data=None, echo='10', msg='SEND_MSG_API_ERROR', retcode=100, status='failed', wording='empty message')

### 预期行为

_No response_

### 实际行为

_No response_

### 日志信息

_No response_

### 额外补充

_No response_

--- a/nonebot_plugin_moyu/__init__.py
+++ b/nonebot_plugin_moyu/__init__.py
@@ -31,7 +31,7 @@ driver = get_driver()
 
 
 async def get_calendar() -> bytes:
-    async with httpx.AsyncClient(http2=True) as client:
+    async with httpx.AsyncClient(http2=True, follow_redirects=True) as client:
         response = await client.get(
             "https://api.j4u.ink/v1/store/other/proxy/remote/moyu.json"
         )

